src/pages/docs/chat/getting-started/kotlin.mdx

```diff
@@ -0,0 +1,1040 @@
+---
+title: "Getting started: Chat with Kotlin"
+meta_description: "A getting started guide for Ably Chat Android that steps through some of the key features using Jetpack Compose."
+meta_keywords: "Ably, realtime, quickstart, getting started, basics, Chat, Android, Kotlin, Jetpack Compose"
+---
+
+This guide will help you get started with Ably Chat in a new Android Kotlin application
+built with Jetpack Compose.
+
+It will take you through the following steps:
+
+* Creating a client and establishing a realtime connection to Ably
+* Creating a room and subscribing to its messages
+* Sending messages to the room and editing messages
+* Retrieving historical messages to provide context for new joiners
+* Displaying online status of clients in the room
+* Subscribing to and sending reactions
+* Disconnecting and resource cleanup
+
+## Prerequisites <a id="prerequisites"/>
+
+### Ably <a id="prerequisites-ably"/>
+
+1. Sign up for an Ably account.
+2. Create a new app and get your API key. You can use the root API key that is provided by default to get started.
+
+3. Install the Ably CLI:
+
+<Code>
+```shell
+npm install -g @ably/cli
+```
+</Code>
+
+4. Run the following to log in to your Ably account and set the default app and API key:
+
+<Code>
+```shell
+ably login
+
+ably apps switch
+ably auth keys switch
+```
+</Code>
+
+<Aside data-type='note'>
+The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI, ensure that you replace them with your own API key.
+</Aside>
+
+### Create a new project <a id="prerequisites-project"/>
+
+Create a new Android project with Jetpack Compose. For detailed instructions, refer to the [Android Studio documentation](https://developer.android.com/jetpack/compose/setup).
+
+1. Create a new Android project in Android Studio.
+2. Select **Empty Activity** as the template
+3. Name the project **Chat Example** and place it in the `com.example.chatexample` package
+4. Set the minimum SDK level to API 24 or higher
+5. Select Kotlin as the programming language
+6. Add the Ably dependencies to your app-level `build.gradle.kts` file:
+
+<Code>
+```kotlin
+implementation("com.ably.chat:chat-android:<latest-version>")
+// This package contains extension functions for better Jetpack Compose integration.
+// It's experimental for now (safe to use, but the API may change later). You can always use its code as a reference.
+implementation("com.ably.chat:chat-extensions-compose:<latest-version>")
+```
+</Code>
+
+## Step 1: Setting up Ably <a id="step-1"/>
+
+Replace the contents of your `MainActivity.kt` file with the following code to set up the Ably client:
+
+<Code>
+```kotlin
```

Comment:
For this to work for me, I had to add `package com.example.chatexample` to the top